### PR TITLE
Pre-packaged revocation list

### PR DESCRIPTION
**Pull request by stmitt**
- store next since along with revocation list (this fixes a potential deviation between next since and the actual revocation list)
- pre-package the current revocation list
- adds unit tests for the revocation list update logic

### CODE REVIEW

File: Sources/CovidCertificateSDK/TrustList/RevocationListUpdate.swift [hunk 1]
```diff
@@ -70,23 +75,12 @@ class RevocationListUpdate: TrustListUpdate {
...
-
-            guard let result = try? outcome.get() else {
+            guard let result = decoder.decode(d) else {
                 return .NETWORK_PARSE_ERROR
             }
 
-            _ = trustStorage.updateRevocationList(result)
-
-            nextSince = nextSinceHeader
+            let success = trustStorage.updateRevocationList(result, nextSince: nextSinceHeader)
```
Comment (UBaggeler) [inline Sources/CovidCertificateSDK/TrustList/RevocationListUpdate.swift:82]: If updating the revocation list fails (for example while saving) do we really want to continue (with the loop)?

Comment (stmitt) [reply Sources/CovidCertificateSDK/TrustList/RevocationListUpdate.swift:82]: The `nextSince` header is stored alongside the actual revocations. Therefore even if the saving fails we still want to fetch the complete revocation list and keep it in memory. On the next app start the list would get fetched again and we would try to save it.

Comment (stmitt) [reply Sources/CovidCertificateSDK/TrustList/RevocationListUpdate.swift:82]: The alternative would be to return a new error case if we are unable to write to the disk.

Comment (UBaggeler) [reply Sources/CovidCertificateSDK/TrustList/RevocationListUpdate.swift:82]: True, I did not remember the in-memory storage. In that case I don't see a reason to change it.

**Pull request merged**